com.tibia.Tibia.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.tibia.Tibia</id>
+
+  <name>Tibia</name>
+  <summary>Become a part of Tibia, the MMORPG classic that has been expanded for more than 25 years</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+
+  <recommends>
+    <display_length compare="ge">1020</display_length>
+  </recommends>
+
+  <description>
+     <p>
+      Tibia is one of the oldest and most successful massively multiplayer online role-playing games (MMORPG) created in Europe. In an MMORPG people from all over the world meet on a virtual playground to explore areas, solve tricky riddles and undertake heroic exploits.
+    </p>
+    <p>
+      For more than 25 years now, players have been visiting the medieval world of Tibia. At present, more than 500,000 players from all over the world inhabit the Tibian continent enjoying the numerous game features.
+    </p>
+    <p>
+      Acting as knights, paladins, sorcerers or druids, players are faced with the challenge of developing the skills of their selected characters, exploring a large variety of areas and dangerous dungeons and interacting with other players on a social and diplomatic level. Besides the sophisticated chat tools it is especially the unique freedom players enjoy in Tibia that create an enormously immersive gaming experience.
+    </p>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported by Cipsoft GmbH.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">com.tibia.Tibia.desktop</launchable>
+
+  <url type="homepage">https://www.tibia.com</url>
+  <url type="faq">https://www.tibia.com/support/</url>
+  <url type="help">https://www.tibia.com/gameguides/</url>
+
+  <developer id="com.cipsoft">
+    <name>Cipsoft GmbH</name>
+    <url type="homepage">https://www.cipsoft.com/</url>
+  </developer>
+
+  <releases>
+    <release version="5cb4ccd3a17b99a571201e92a8f925f5f19e6c63" date="2024-09-12"/>
```

Comment:
How is the version a string ? Or is this a commit ?

Reply:
I don't know how their versioning works, it is not sequential and is available in the link I added in the description. Probably an internal repository commit or checksum.

They update the launcher every once in a while but there are no public version numbers. They don't separate launcher and client update news.

This is the _client_ version (which is downloaded by the launcher, this package: https://static.tibia.com/launcher/tibiaclient-linux-current/package.json.version)

Sorry if this is confusing, I don't get it either 😆